package.json

```diff
@@ -37,9 +37,9 @@
     "chai-as-promised": "^5.3.0",
     "devtron": "^1.3.0",
     "electron-builder": "5.2.1",
-    "electron-connect": "~0.3.9",
-    "electron-packager": "^7.0.1",
-    "electron-prebuilt": "1.2.8",
+    "electron-connect": "~0.6.0",
+    "electron-packager": "^8.0.0",
+    "electron-prebuilt": "1.4.0",
```

Comment:
I have checked [a known issue](https://github.com/mattermost/desktop/pull/214#issuecomment-237263102) with servers, pre-release and v3.2.0. The issue doesn't happen. Did you check?
Then, to make sure, we might have to write its compatibility and workaround in the changelog. Any thoughts?